tests/iot_config.h

```diff
@@ -223,13 +275,13 @@
      * printed. */
     #define IotLogging_Puts       _coveragePuts
 
-    /* Includes for coverage logging puts. */
+/* Includes for coverage logging puts. */
```

Comment:
This indentation change seems off.

Reply:
This is the formatting produced by Uncrustify-0.68-2-8c80bd84 on Linux..